src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6833,32 +6832,29 @@ public override IOperation VisitSwitchExpression(ISwitchExpressionOperation oper
             return GetCaptureReference(captureOutput, operation);
         }
 
-        private void VisitUsingVariableDeclarationOperation(VariableDeclarationGroupOperation operation, ImmutableArray<IOperation> statements)
+        private void VisitUsingVariableDeclarationOperation(IUsingDeclarationOperation operation, ImmutableArray<IOperation> statements)
         {
             IOperation saveCurrentStatement = _currentStatement;
-            _currentStatement = operation;
-            StartVisitingStatement(operation);
-
-            var declarationKind = operation.DeclarationKind;
-            Debug.Assert(declarationKind == VariableDeclarationKind.Using || declarationKind == VariableDeclarationKind.AsynchronousUsing);
+            _currentStatement = operation.DeclarationGroup;
```

Comment:
>_currentStatement = operation.DeclarationGroup; [](start = 12, length = 47)

We probably should stick to the old behavior here, ```IUsingDeclarationOperation``` is at the statement position.

Reply:
The reason I didn't do this is because `HandleUsingOperationParts` will eventually call to the base `VisitVariableDeclarationGroup`, which assumes the declaration group is the current statement. We'd either have to special case in this method, or special case in the Visit method to look at the parent, which feels somewhat fragile.

---
In reply to: [340367372](https://github.com/dotnet/roslyn/pull/39576#discussion_r340367372) [](ancestors = 340367372)

Reply:
I'll add a comment to explain this.

---
In reply to: [340368727](https://github.com/dotnet/roslyn/pull/39576#discussion_r340368727) [](ancestors = 340368727,340367372)